src/operator/nn/mkldnn/mkldnn_pooling-inl.h

```diff
@@ -92,6 +92,8 @@ inline bool SupportMKLDNNPooling(const PoolingParam &param,
 
   if (param.pooling_convention == pool_enum::kValid)
     return true;
+  else
+    return false;
```

Comment:
this makes the later code unreachable. Are you sure this is what you want?

Reply:
@piiswrong the later code is being disabled for now until we support all "full" pooling convention cases.  As @TaoLv mentioned, i will add a unit-test for this failure. 
We may refactor this code, so marking this PR WIP for now.

Reply:
maybe use "#if 0" to comment the checks below?